modules/tinymce/src/models/dom/test/ts/module/table/TableTestUtils.ts

```diff
@@ -53,7 +53,7 @@ const assertWidth = (editor: Editor, elm: HTMLElement, expectedWidth: number | n
   if (expectedWidth === null) {
     assert.isNull(widthData.raw, `${nodeName} width should not be set`);
   } else {
-    assert.approximately(widthData.raw ?? -1, expectedWidth, 2, `${nodeName} width is ${expectedWidth} ~= ${widthData.raw}`);
+    assert.approximately(widthData.raw ?? -1, expectedWidth, 4, `${nodeName} width is ${expectedWidth} ~= ${widthData.raw}`);
```

Comment:
This looks weird. I guess the width is off by 1 pixel in both right and left, which gives us off by 2 pixels. Right ?

Reply:
Maybe just put a comment explaining why `4` has been chosen (even if it's just: that's what it seemed to be based on actual data)

Reply:
It didn't have a comment for 2 either. Not 100% sure why 4 is needed might need some heavy investigation why the Firefox now is off by 2 more pixels. So not sure at this time what to put into the comment.

Reply:
K. Just putting something like "this 4 seems to be large enough to handle the different values we are seeing, but it's never been thoroughly investigated"

Reply:
I added a comment and referencing the ticket added information on what failed and the weirdness around that in the JIRA ticket instead.